packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -552,6 +616,40 @@ contract Deploy is Deployer {
         }
     }
 
+    /// @notice Initialize the Registry
+    function initializeRegistry() public broadcast {
+        ProxyAdmin proxyAdmin = ProxyAdmin(mustGetAddress("ProxyAdmin"));
+        address registryProxy = mustGetAddress("RegistryProxy");
+        address registry = mustGetAddress("Registry");
+
+        proxyAdmin.upgradeAndCall({
+            _proxy: payable(registryProxy),
+            _implementation: registry,
+            _data: abi.encodeCall(
+                Registry.initialize,
+                ()
+                )
+        });
+    }
+
+    /// @notice Initialize the GoldToken
+    function initializeGoldToken() public broadcast {
+        ProxyAdmin proxyAdmin = ProxyAdmin(mustGetAddress("ProxyAdmin"));
+        address goldTokenProxy = mustGetAddress("GoldTokenProxy");
+        address goldToken = mustGetAddress("GoldToken");
+
+        address registryProxy = mustGetAddress("RegistryProxy");
+
+        proxyAdmin.upgradeAndCall({
+            _proxy: payable(goldTokenProxy),
+            _implementation: goldToken,
+            _data: abi.encodeCall(
+                GoldToken.initialize,
+                (registryProxy)
+                )
+        });
```

Comment:
If we can call the proxyAdmin here, can we also call the Registry to register our GoldToken right away?